docs/api/users-update-by-id.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+---
+
+# Update user by ID
+
+Allows the editing of [`user`](user) objects that already exist. You will need to know the `id` parameter of the user you want to edit.
+
+> [!IMPORTANT]
+> The information for the user is replaced **entirely** by the request body, so if you want to keep information the same you still need to include it in the request body. The only exception to this is the ID which remains the same regardless of the body that is submitted.
+
+## HTTP Method
+PUT
+
+## URL
+
+```shell
+{server_url}/users/{id}
+```
+
+## Params
+
+| Parameter name | Type | Description |
+| -------------- | ------ | ------------ |
+| `id` | number | The record ID of the user to return |
+
+## Request headers
+
+| Key | Value |
+|---|---|
+| Content-Type | application/json |
+
+## Request body parameters
+
```

Comment:
Maybe add a note that describes that all properties (except for the `id`) are required and describe what happens if a property is omitted. (per comment above)